rhine/src/FRP/Rhine/Clock/Select.hs

```diff
@@ -38,15 +39,15 @@ instance (Monad m, Clock m cl) => Clock m (SelectClock cl a) where
 
 
 -- | A universal schedule for two subclocks of the same main clock.
---   The main clock must be a monoid (e.g. a singleton).
+--   The main clock must be a Semigroup (e.g. a singleton).
```

Comment:
Ideally, make a haddock link, i.e. `'Semigroup'`

Reply:
Cool ok! Should I do the PR again, or is this just for future reference?

Reply:
For the real experience, push one commit to your fork that fixes it!

Reply:
haha ok :)